coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -33,3 +41,107 @@ def __init__(self, documentation, language, docstyle,
 
     def __str__(self):
         return self.documentation
+
+    def parse_documentation(self):
+        """
+        Parses documentation independent of language and docstyle.
+
+        :return:
+            A list of the parsed documentation metadata.
+        :raises:
+            NotImplementedError
+        """
+        if self.language == "python":
+            return self.parse_python()
+        else:
+            raise NotImplementedError(
+                "Documentation parsing for {} in {}"
+                " has not been implemented yet".format(
+                    self.language, self.docstyle))
```

Comment:
let's do it cool :D
`"Documentation parsing for {0.language} in {0.docstyle} has not been implemented yet".format(self)`